GOALS.md

```diff
@@ -25,4 +25,4 @@ The goals of this release are as follows:
        domains and arrays using some standard distributions.
 
 For more detailed notes on what is or is not supported in this
-release, refer to the STATUS file in this directory.
+release, refer to the Issues list on GitHub.
```

Comment:
Tell them where the Issues list on GitHub is? (https://github.com/chapel-lang/chapel/issues)